modules/features/search/src/main/java/au/com/shiftyjelly/pocketcasts/search/SearchFragment.kt

```diff
@@ -244,15 +251,21 @@ class SearchFragment : BaseFragment() {
         binding.searchInlineResults.apply {
             ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed
             setContent {
+                val bottomInset by settings.bottomInset.collectAsStateWithLifecycle(initialValue = 0)
+                val state by viewModel.state.collectAsState()
+                val loading by viewModel.loading.asFlow().collectAsState(false)
                 AppThemeWithBackground(theme.activeTheme) {
                     SearchInlineResultsPage(
-                        viewModel = viewModel,
+                        state = state,
+                        loading = loading,
                         onEpisodeClick = ::onEpisodeClick,
                         onPodcastClick = ::onPodcastClick,
                         onFolderClick = ::onFolderClick,
                         onShowAllCLick = ::onShowAllClick,
+                        onSubscribeToPodcast = ::onSubscribeToPodcast,
```

Comment:
I also cleaned up this code by lifting the view model calls out of the component.